recipes-wpe/wpeframework/include/firmwarecontrol.inc

```diff
@@ -0,0 +1,21 @@
+WPE_FIRMWARECONTROL_EXTRAFLAGS += "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/mfrlib', 'mfrlibs-rpi', '-DPLUGIN_FIRMWARECONTROL_PLATFORM_RPI=ON', '', d)}"
+
+# Provider for RPI platform has to be updated once the implementation for mfr fwupgrade is available
+RDEPENDS_${PN} += "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/mfrlib', 'mfrlibs-rpi', 'mfrlibs-rpi', '', d)}"
+RDEPENDS_${PN} += "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/mfrfwlib', 'fwupgrade-lib', 'fwupgrade-lib', '', d)}"
+
+# Additional flags
+WPE_FIRMWARECONTROL_AUTOSTART ?= "true"
+# ----------------------------------------------------------------------------
+WPE_FIRMWARECONTROL_EXTRAFLAGS += ' \
+-DPLUGIN_FIRMWARECONTROL_AUTOSTART=${WPE_FIRMWARECONTROL_AUTOSTART} \
+'
```

Comment:
No need for an extra flags structure, you can just add `WPE_FIRMWARECONTROL_AUTOSTART` to the PACKAGE CONFIG below. Unless you expect additional flags to be needed?

Reply:
will update

Reply:
OK